application/error_handlers.py

```diff
@@ -0,0 +1,21 @@
+import flask
+from flask import jsonify
+from requests import HTTPError
+
+
+blueprint = flask.Blueprint('error_handlers', __name__)
+
+
+@blueprint.app_errorhandler(Exception)
+def exception_error(error):
+    response = jsonify({'errors': error if type(error) is list else [error]})
```

Comment:
I don't think this will ever be a list. It will be either a code or exception but it will be an exception in this case. It's unlikely we want to respond with the traceback. We should be logging and returning 500